deployment.yaml

```diff
@@ -38,13 +40,7 @@ spec:
           value: "3500"
         image: sensrnetregistry.azurecr.io/sensrnet/sync-bridge:latest
         name: sync-bridge
-        volumeMounts:
```

Comment:
Hmm... ik verwacht dat we secrets moeten behouden? Verderop staan de username/pw hardcoded in de config 🙈

Reply:
Ik wist niet zeker of deze deployment.yaml ook echt gebruikt werd, ik zag geen soortgelijke deployment in de backend repo. Secrets lijken mij ook prima, al gaat het om de username / password om met de blockchain node binnen de cluster te communiceren, waar je alleen binnen de cluster bij kan komen (RPC poorten niet open zetten naar buiten), dus niet perse noodzakelijk. We kunnen dat ook later oppakken.

Reply:
OK. Lijkt me goed om dat later wel beter beveiligd te maken. Alleen toegang binnen het cluster is niet veilig genoeg. Wel voor dit moment 😉